### ISSUE TITLE
Idle Leveling Up?

### ISSUE THREAD

**Issue by lskitto** [mcMMO-Dev/mcMMO#1955]
This is not a bug (well could be) but something rather odd (and cool)!
Well, I went on my server (play.craft101.info) and stood idle for some mins (not sure exact but, 15m I think), and browsed on my other computer (Facebook), forgot about Minecraft on the previous computer and when I glanced over, it said "Flying skill increased by 1", but I was at SPAWN the whole time, in SURVIVAL, and FULLY on the GROUND? Amazing!

Basically:
1. Logged into my server
2. Idle for some mins
3. "Flying skill increased by 1"
